medmodels/treatment_effect/continuous_estimators.py

```diff
@@ -50,46 +51,76 @@ def average_treatment_effect(
             exposure time. Options include "first" and "last". If "first", the function
             returns the earliest exposure edge. If "last", the function returns the
             latest exposure edge. Defaults to "last".
-        time_attribute (MedRecordAttribute, optional): The attribute in the edge that
-            contains the time information. Defaults to "time".
+        time_attribute (Optional[MedRecordAttribute], optional): The attribute in the
+            edge that contains the time information. If it is equal to None, there is
+            no time component in the data and all edges between the sets and the
+            outcomes are considered for the average treatment effect. Defaults to
+            "time".
 
     Returns:
         float: The average treatment effect.
 
     Raises:
         ValueError: If the outcome variable is not numeric.
     """
-    treated_outcomes = np.array(
-        [
-            medrecord.edge[
-                find_reference_edge(
-                    medrecord,
-                    node_index,
-                    outcome_group,
-                    time_attribute=time_attribute,
-                    reference=reference,
-                )
-            ][outcome_variable]
-            for node_index in treatment_outcome_true_set
-        ]
-    )
+    if time_attribute is not None:
+        treated_outcomes = np.array(
+            [
+                medrecord.edge[
+                    find_reference_edge(
+                        medrecord,
+                        node_index,
+                        outcome_group,
+                        time_attribute=time_attribute,
+                        reference=reference,
+                    )
+                ][outcome_variable]
+                for node_index in treatment_outcome_true_set
+            ]
+        )
+        control_outcomes = np.array(
+            [
+                medrecord.edge[
+                    find_reference_edge(
+                        medrecord,
+                        node_index,
+                        outcome_group,
+                        time_attribute="time",
+                        reference=reference,
+                    )
+                ][outcome_variable]
+                for node_index in control_outcome_true_set
+            ]
+        )
+
+    else:
+        edges_treated_outcomes = medrecord.select_edges(
+            lambda edge: query_edges_set_outcome(
+                edge, treatment_outcome_true_set, outcome_group
+            )
+        )
+        treated_outcomes = treated_outcomes = np.array(
```

Comment:
```suggestion
        treated_outcomes = np.array(
```

is this maybe a typo or what does that mean?